lib/index.js

```diff
@@ -104,14 +116,21 @@ class EmailValidator {
     return records.map(record => record.exchange);
   }
 
-  static async verifyMailbox(local, domain, [mxRecord], timeout, log, fromAddress) {
-    if (!mxRecord || /yahoo/.test(mxRecord)) {
+  static async verifyMailbox(local, domain, [mxRecord], timeout, log, fromAddress, useOpenSSL, portList) {
+    if (/yahoo/.test(mxRecord)) {
+      log.debug('Yahoo MX record not supported', mxRecord);
```

Comment:
Its not that we don't support the yahoo MX record, it is just that yahoo doesn't tell us if an email address exists when asked (via smtp)